utils/virtcontainers-setup.sh

```diff
@@ -42,16 +39,16 @@ virtcontainers_build_dir="virtcontainers/build"
 echo -e "Create temporary build directory ${tmpdir}/${virtcontainers_build_dir}"
 mkdir -p ${tmpdir}/${virtcontainers_build_dir}
 
-TMPDIR="/tmp"
+TMPDIR="${SCRIPT_PATH}/supportfiles"
```

Comment:
Could you add a commit adding this whole directory to `.gitignore` ? This way, we won't have git tracking those files that we could add to a patch by mistake.

Reply:
(oops, I meant to but forgot...) done and pushed